recipe/meta.yaml

```
@@ -34,8 +34,10 @@ outputs:
         - libtool
         - make
         - pkg-config
+        - rdma-core =28.*  # Use version 28.* to supports MOFED 5.0.x and above
       host:
         - libnuma
+        - rdma-core =28.*  # Use version 28.* to supports MOFED 5.0.x and above
       run:
```

```suggestion
      run:
        - {{ pin_subpackage('rdma-core') }}
```

Note: This shouldn't be needed once a new version of `rdma-core` is packaged with `run_exports` ( https://github.com/conda-forge/rdma-core-feedstock/pull/3 )

---

We already have a new version so I intentionally skipped this one.

---

Would suggest keeping it since we only have a new `build/number` (not a new `version`) and conda could choose the older `build/number` package, which would leave this unconstrained

Alternatively we could try [marking the old `build/number` packages broken]( https://github.com/conda-forge/admin-requests#mark-packages-as-not-broken-on-conda-forge )